linux/system/kernel.sls

```diff
@@ -53,6 +53,17 @@ linux_kernel_module_{{ module }}:
 
 {%- endfor %}
 
+{%- for kmod_config, kmod_content in system.kernel.get('kmod_config', {}).iteritems() %}
+
+/etc/modprobe.d/{{ kmod_config }}.conf:
```

Comment:
state before this one loads kernel modules, so if one to apply configuration for any module, he should make sure that it is loaded by default, or load it using state inux_kernel_module_{{ module }}, that is just 2 lines above